docs/getting-started.md

```diff
@@ -391,4 +394,121 @@ curl -H "Authorization: Bearer {Key ID}" localhost:8080/httpbin/get
 }
 ```
 
-Since we have provided a valid key along with our request, we do not have `HTTP 401 Unauthorized` response. 
\ No newline at end of file
+Since we have provided a valid key along with our request, we do not have `HTTP 401 Unauthorized` response. 
+
+
+
+
+## Publish an API to the Tyk Portal
+
+Asumming that you have created the httpbin example API using the steps above, we can easily publish it to the Tyk Portal by applying a few specs:
+1. SecurityPolicy spec
+2. APIDescription spec
+3. PortalAPICatalogue spec
+
+
+### 1. Adding a security policy via Tyk Operator
+
+When you publish an API to the Portal, Tyk actually publishes a way for developers to enrol in a policy, not into the API directly. Therefore, we must also apply a SecurityPolicy CRD before proceeding with the publishing.
+
+For this we can create a file called "securityPolicyHttpBin.yaml" with the following contents :
```

Comment:
It might be better to use backticks (**\`**) to indicate `"securityPolicyHttpBin.yaml"`. So, it will be highlighted by the markdown highlighter.